Makefile

```diff
@@ -58,6 +58,11 @@ checksum:
 clean:
 	@rm -rf $(BINDIR) ./_dist
 
+dev:
+	go generate $(GOFILES)
```

Comment:
Ah I see,

So the `generate` and `build` targets to thing I didn't wan't in my dev target:

Primarily `-v`